common/views/components/IIIFViewer/IIIFViewer.js

```diff
@@ -0,0 +1,339 @@
+      left: 50%;
+      transform: translateX(-50%) translateY(0%);
+
+      .control__inner {
+        transform: rotate(90deg);
+      }
+    }
+  `}
+`;
+
+const IIIFViewerThumbNumber = styled.span.attrs(props => ({
+  className: classNames({
+    'line-height-1': true,
+    'absolute bg-charcoal font-white': true,
+    [font({ s: 'LR3' })]: true,
+  }),
+}))`
+  top: ${props => props.theme.spacingUnit}px;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: 3px 2px 0;
+  z-index: 1;
+`;
+
+const IIIFViewerThumb = styled.div.attrs(props => ({
+  className: classNames({
+    'relative flex flex--v-center': true,
+    [spacing({ s: 1 }, { padding: ['top', 'right', 'bottom', 'left'] })]: true,
+  }),
+}))`
+  height: 100%;
+  width: 20%;
+  margin-right: ${props => props.theme.spacingUnit}px;
+
+  &:last-child {
+    margin: 0;
+  }
+
+  @media (min-width: ${props => props.theme.sizes.medium}px) {
+    height: 25%;
+    width: 100%;
+    margin-right: 0;
+  }
+`;
+
+const IIIFViewerThumbs = styled.div.attrs(props => ({
+  className: classNames({
+    'flex flex--h-center relative bg-smoke': true,
+  }),
+}))`
+  height: 20%;
+  width: 100%;
+  padding: 0 100px 0 0;
+
+  @media (min-width: ${props => props.theme.sizes.medium}px) {
+    height: 100%;
+    flex-direction: column;
+    width: 25%;
+    padding: 0 0 ${props => props.theme.spacingUnit * 10}px;
+  }
+`;
+
+const IIIFViewerMain = styled.div.attrs(props => ({
+  className: classNames({
+    'relative bg-charcoal font-white': true,
+    [spacing({ s: 4 }, { padding: ['top'] })]: true,
+    [spacing({ s: 1 }, { padding: ['right', 'left'] })]: true,
+    [spacing({ s: 10 }, { padding: ['bottom'] })]: true,
+  }),
+}))`
+  height: 80%;
+  width: 100%;
+
+  @media (min-width: ${props => props.theme.sizes.medium}px) {
+    height: 100%;
+    width: 75%;
+  }
+`;
+
+const IIIFViewerXOfY = styled.span.attrs(props => ({
+  className: classNames({
+    'absolute font-white': true,
+    [spacing({ s: 1 }, { margin: ['left', 'right'] })]: true,
+    [font({ s: 'LR3' })]: true,
+  }),
+}))`
+  top: ${props => props.theme.spacingUnit}px;
+  left: 50%;
+  transform: translateX(-50%);
+`;
+
+const IIIFViewerThumbLink = styled.a.attrs(props => ({
+  className: classNames({
+    'block h-center': true,
+  }),
+}))`
+  height: 100%;
+
+  img {
+    border: 3px solid
+      ${props => (props.isActive ? props.theme.colors.white : 'transparent')};
+    transition: border-color 200ms ease;
+  }
+`;
+
+const IIIFViewer = styled.div.attrs(props => ({
+  className: classNames({
+    'flex flex--wrap': true,
+  }),
+}))`
+  width: 100vw;
+  height: 90vh;
+  flex-direction: row-reverse;
+
+  img {
+    position: relative;
+    top: 50%;
+    transform: translateY(-50%);
+    margin: 0 auto;
+    display: block;
+    width: auto;
+    height: auto;
+    max-width: 100%;
+    max-height: 100%;
+    overflow: scroll;
+  }
+`;
+
+type IIIFCanvasThumbnailProps = {|
+  canvas: IIIFCanvas,
+  maxWidth: ?number,
```

Comment:
Is this nullable?

Reply:
Or for that matter even used in the component?

Reply:
Good point.